packages/cozy-notifications/src/urls.js

```diff
@@ -0,0 +1,57 @@
+// Copied from ui/react/AppLinker/native
```

Comment:
and edited 

Can you send your edit upstream?

Reply:
PR in UI: https://github.com/cozy/cozy-ui/pull/1172

I still have to copy the file in cozy-notifications since :

1. cozy-ui is shipped with es6 modules, that makes it unsuitable for consumption in connectors (for now, we have to discuss on how to remedy that)

2. the universalLink mechanism relies on overriding the value at build time but we will not do that for konnectors

I made an issue here : https://github.com/cozy/cozy-ui/issues/1173